src/views/ProductPage.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div>
+    <div class="container">
+      <div class="col-12">
+        <nav class="navbar navbar-light bg-white">
+          <a href="#" @click="$router.go(-1)">&lt; Tilbage til søgningen</a>
```

Comment:
This will not work in all cases but we can keep it for now as we are going to revamp the whole thing soon